draft-issuance.md

```diff
@@ -0,0 +1,166 @@
+```
+ZIP: 
+Title: Smooth Out The Block Subsidy Issuance
+Owners: Jason McGee <jason@shieldedlabs.com>
+        Mark Henderson <mark@equilibrium.co>
+        Tomek Piotrowski <tomek@eiger.co>
+        Mariusz Pilarek <mariusz@eiger.co>
+Original-Authors: Nathan Wilcox
+Credits: Nathan Wilcox
+         Mark Henderson
+         Jason McGee
+Status: Draft
+Category: Consensus
+Created: 2023-08-23
+License: BSD-2-Clause
+```
+
+# Terminology
+
+The key words “MUST”, “SHOULD”, “SHOULD NOT”, “MAY”, “RECOMMENDED”, “OPTIONAL”,
+and “REQUIRED” in this document are to be interpreted as described in RFC 2119. [1]
+
+"Network upgrade" - to be interpreted as described in ZIP 200. [2]
+
+“Block Subsidy” - the algorithmic issuance of ZEC on block creation – part of
+the consensus rules. Split between the miner and the Dev Fund. Also known as Block Reward.
+
+“Issuance” - The method by which unmined or unissued ZEC is converted to ZEC available
+to users of the network
```

Comment:
These definitions are repeated in PRs #703 and #718, please define the terms in the first ZIP #703, then reference them from the other ZIPs.

I suggest using moving these definitions to #703, because they are correct. The definitions in the other PRs are slightly different in a way that makes them incorrect.

Reply:
These now reference the ZIP in #703